Reframe Cloud vs MST

charislam opened the pull request:
Make it clearer what the distinction is between Cloud and MST, and how
readers should choose.

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

charislam committed:
Reframe Cloud vs MST

Make it clearer what the distinction is between Cloud and MST, and how
readers should choose.

charislam commented:
Changed pages for easy preview: 

- http://docs-dev.timescale.com/docs-charis-cloud-mst/ (look at the featured cards)
- http://docs-dev.timescale.com/docs-charis-cloud-mst/install/charis-cloud-mst/installation-mst/
- http://docs-dev.timescale.com/docs-charis-cloud-mst/mst/charis-cloud-mst/
- http://docs-dev.timescale.com/docs-charis-cloud-mst/mst/charis-cloud-mst/about-mst/

charislam committed:
Add updates to product FAQ

Loquacity commented on _partials/_cloud-mst-comparison.mdx:
```diff
@@ -0,0 +1,11 @@
+  we recommend Timescale Cloud, which we designed specifically for TimescaleDB.
+  On Azure, GCP, and some less common regions of AWS, we recommend our partnered
+  service, Managed Service for TimescaleDB. [See the available Timescale Cloud
+  regions.](/cloud/latest/#available-in-multiple-aws-regions)
```
I'm not 100% convinced that this is the right link. It feels a little jarring to be like "here's how to choose" then straight into "Cloud regions". Maybe we need to have a page that goes into a little more detail (I mean, there are significant feature differences between the two as well, it's not just regions)? Or link to MST regions as well? I'm not sure ...

Rajakavitha1 replied:
Hi @charislam  and @Loquacity  Here's what Alex shared with me 
<img width="988" alt="Screenshot 2022-08-29 at 10 34 23 AM" src="https://user-images.githubusercontent.com/37059749/187126871-978dc48f-3f9d-44c0-8168-4bf733f2aa5b.png">

charislam replied:
Yes I agree it might be a bit abrupt. The purpose is to simplify the "how to choose." Clouds and regions is the major factor and is how our signup wizard works as well. But I'm going to fine-tune this a bit to make it less jarring!

charislam committed:
Fine-tune language in Cloud callout

Loquacity reviewed (approved):
This looks better to me 👍

carlotas19 commented:
Just a side note, everyone:The exec team is working on new positioning pillars for Timescale Cloud, so the copy we use here should reflect that 👌

I'll send over some edits soon (probably tomorrow!)

charislam committed:
Update positioning for Timescale Cloud and MST

Update positioning of Timescale Cloud and MST in the about,
installation, and FAQ pages.
